### PR TITLE
MINOR: Additional fix on the rollback migration documentation

### PR DESCRIPTION
**Pull request by ppatierno**
This is an additional PR to fix rollback migration documentation related to [#15287](https://github.com/apache/kafka/pull/15287)

### CODE REVIEW

**Comment by ppatierno** [comment]
@cmccabe @showuon can you take a look at this please?

File: docs/ops.html [hunk 1]
```diff
@@ -4054,16 +4054,18 @@ <h3>Reverting to ZooKeeper mode During the Migration</h3>
           </ul>
         </td>
         <td>
-          It is important to perform the <code>zookeeper-shell.sh</code> step quickly, to minimize the amount of
-          time that the cluster lacks a controller.
+          It is important to perform the <code>zookeeper-shell.sh</code> step <b>quickly</b>, to minimize the amount of
+          time that the cluster lacks a controller. Until the <code> /controller</code> znode is deleted,
+          you can also ignore any broker errors in the log about controller missing.
```
Comment (showuon) [inline docs/ops.html]: > Until the <code> /controller</code> znode is deleted, you can also ignore any broker errors in the log about controller missing.

Maybe:
`You can also ignore any errors in the broker log about failing to connect to the Kraft controller. Those error logs should disappear after second roll to pure zookeeper mode.`

WDYT?

Comment (ppatierno) [reply docs/ops.html]: I was avoiding too much text, but of course this one is clearer. I will update it.

Comment (ppatierno) [reply docs/ops.html]: Done.

File: docs/ops.html [hunk 2]
```diff
@@ -4054,16 +4054,19 @@ <h3>Reverting to ZooKeeper mode During the Migration</h3>
           </ul>
         </td>
         <td>
-          It is important to perform the <code>zookeeper-shell.sh</code> step quickly, to minimize the amount of
-          time that the cluster lacks a controller.
+          It is important to perform the <code>zookeeper-shell.sh</code> step <b>quickly</b>, to minimize the amount of
+          time that the cluster lacks a controller. Until the <code> /controller</code> znode is deleted,
```
Comment (showuon) [inline docs/ops.html:4058]: `Until the <code> /controller</code> znode is deleted, you can also ignore any errors in the broker log`

From my understanding, I think the error will disappear after broker roll into pure ZK mode, that is, the broker config doesn't contain voters info. Is that right?

Comment (ppatierno) [reply docs/ops.html:4058]: Yes. This is what I am saying in the next sentence `Those error logs should disappear after second roll to pure zookeeper mode.`

**Review by showuon — APPROVED** [review]
LGTM!

**Pull request merged**